pkg/karmadactl/apply/apply.go

```diff
@@ -26,7 +26,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/cli-runtime/pkg/genericclioptions"
```

Comment:
Do you mean this package has been deprecated?
How do you know it?

Reply:
Not all deprecated, we can see the relevant information from this commit.
https://github.com/kubernetes/cli-runtime/commit/b617b8621b3b1b80198f2185b241240448dca599